src/docs/tutorial-branding/modify-landing-page/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Modify your landing page
+slug: modify-landing-page
+updated: 2023-01-16
+category: tutorial-branding
+ingress: Change the look of your landing page
+published: true
+---
+
+The default landing page for the Saunatime template shows generic
+instructions and images for building a successful landing page. In this
+tutorial, you will learn to modify the **hosted assets** on your landing
+page.
+
+![Default landing page](./default-landing-page.png)
+
+You can modify your landing page in Flex Console > Build > Content. By
+default, your marketplace has four content pages already created:
+
+- Landing page
+- Terms of Service
+- Privacy Policy
+- About
+
+![Default pages](./default-pages.png)
+
+Each page is identified by their page id. For pages other than landing
+page, the page id is reflected in the page address on your marketplace.
+The landing page, however, shows up on the root page of your client site
+by default.
+
+For each page, you can see two links: _Edit content_ and _View page_. To
+edit a page, you can either click the link, or just click anywhere in
+the white container for the page.
+
+Once you click the link, you can see the default **sections** for the
+landing page. You can freely modify, add, and remove the sections as you
+develop your marketplace.
+
+In this tutorial, you will modify the first section, _hero_. Click on
+the section to open the details.
+
+<video>
+    <source src='./lpsh.mp4' type='video/mp4'>
+    <source src='./lpsh.webm' type='video/webm'>
+    <source src='./lpsh.ogv' type='video/ogg'>
+</video>
+
+### Update section background image
+
+With the hero section open, scroll down to the input for _Background
+image_.
+
+![Landing page image inputs](./landing-page-image-inputs.png)
+
+Remove the previous one, and then upload the same background image we
+used in the previous step for Cottage Days: <br />
+
+- [Summer house by Markus Spiske (cropped)](/tutorial-assets/markus-spiske-summer-house-unsplash.jpg)
+
+With this image, we also need to switch the text from light to dark. To
```

Comment:
The data has now changed in the Console: it's **white** or **black**

In templates, if `textColor: 'white' that value triggers darkTheme mode (text is white, links are underlined, and the background is expected to be dark (bg-color or bg-image) - but that's subject to content writer to handle through Console.